src/Illuminate/Support/Arr.php

```diff
@@ -160,24 +160,38 @@ public static function exists($array, $key)
      * @return mixed
      */
     public static function first($array, callable $callback = null, $default = null)
+    {
+        $key = static::firstKey($array, $callback);
+
+        return is_null($key) ? $default : static::get($array, $key);
```

Comment:
shouldn't you add the `$default` to the `::get()` method?

Reply:
Well, if `firstKey` does not return null, the value must exist and therefore the `$default` will never be used. On the other hand, if the `$key` is null, `::get()` would return the whole `$array`.